index.js

```diff
@@ -87,15 +94,7 @@ function coffeeify(filename, options) {
         header: false
     };
 
-    for (var i = 0, keys = Object.keys(compileOptions); i < keys.length; i++) {
```

Comment:
This to me was way over engineered here. The options to be fancy to support strings to bools and handle 'false', no or whatever to me is just not javascript. This was limiting the ability to add more options to our sourceMap settings without the need for another flag. 

If your totally against this I can revert it but to me the new change is much simpler. Just use booleans unless something special is explicitly supported. Otherwise to make sourceMap work we will need to black list as a boolean only.